pandas/formats/style.py

```diff
@@ -10,7 +10,9 @@
 from collections import defaultdict, MutableMapping
 
 try:
-    from jinja2 import Template
+    from jinja2 import (
+        PackageLoader, Environment, ChoiceLoader, FileSystemLoader
+    )
 except ImportError:
     msg = "pandas.Styler requires jinja2. "\
```

Comment:
do 

```
try:
     from jinja2 .....
except ImportError:
     class Styler(ImportError):
            pass
     return
```

will give you a ``Styler`` but if you try to use it and ``jinja2`` is not installed will raise I think you can give it a nice message a well.